### PR TITLE
Add search options to support deterministic result for axiomatic reranking

### PR DESCRIPTION
**Pull request by Peilin-Yang**
This is critical for reproducibility

### CODE REVIEW

**Comment by Peilin-Yang** [comment]
Waiting for #269 to be merged first.

File: src/main/java/io/anserini/search/SearchArgs.java [hunk 1]
```diff
@@ -84,6 +84,15 @@
   @Option(name = "-axiom", usage = "use Axiomatic query expansion model for the reranking")
   public boolean axiom = false;
 
+  @Option(name = "-axiom.decisive", usage = "decisive the expansion terms axiomatic reranking")
```
Comment (lintool) [inline src/main/java/io/anserini/search/SearchArgs.java]: `-axiom.deterministic` sounds more accurate?

Comment (lintool) [reply src/main/java/io/anserini/search/SearchArgs.java]: Also, you don't want the seed to be settable in the cmd line?

Comment (Peilin-Yang) [reply src/main/java/io/anserini/search/SearchArgs.java]: Well, I just feel nobody is going to use it unless for deterministic results.... 
But I can add it anyway.

**Comment by lintool** [comment]
@Peilin-Yang Good for merging.

Are you planning on updating the documentation?
What about regression tests (bm25+ax, ql+ax)?

**Pull request merged**